libbeat/docs/shared-beats-attributes.asciidoc

```diff
@@ -3,6 +3,8 @@
 :dockerimage: docker.elastic.co/beats/{beatname_lc}:{version}
 :dockerconfig: https://raw.githubusercontent.com/elastic/beats/{branch}/deploy/docker/{beatname_lc}.docker.yml
 :downloads: https://artifacts.elastic.co/downloads/beats
+:libbeat-processors-dir: {beats-root}/libbeat/processors
+:filebeat-processors-dir: {beats-root}/x-pack/filebeat/processors
```

Comment:
I wonder how painful it will be if we have to create attributes for a many more directories in the future. Like inputs, modules, outputs.

lets rename `filebeat-processors-dir` to `xpack-filebeat-processors-dir: {xpack-root}/filebeat/processors`.

I wonder if we even need the attribute, given the attribute reflect the complete path.

Reply:
++ on renaming. 

I think the attributes make it easier to move stuff around later on and they allow someone consuming the content to override the path, if necessary, when including files. Adding them is fairly easy.